src/OSPSuite.Presentation/Presenters/Importer/ImporterDataPresenter.cs

```diff
@@ -83,15 +81,16 @@ public void ImportDataForConfirmation(string sheetName)
             Sheets.Add(sheetName, getSingleSheet(sheetName));
             sheets.Add(sheetName, getSingleSheet(sheetName));
          }
-         if (sheets.Count == 0) 
-            return;
 
-         OnImportSheets.Invoke(this, new ImportSheetsEventArgs { DataSourceFile = _dataSourceFile, Sheets = sheets, Filter = GetActiveFilterCriteria()});
+         if (sheets.Count == 0) return;
```

Comment:
Why are all those return on one line now? Is your resharper config doing this? Can you use the one I sent you a while back? i can definitely sent it back to you if needed